ingestion/src/metadata/ingestion/source/database/vertica/metadata.py

```diff
@@ -108,8 +108,9 @@ def get_columns(
             comment,
         )
         column_info.update({"primary_key": primary_key})
-        columns.append(column_info)
-    return columns
+        if columns.get(name) is None or comment:
+            columns[name] = column_info
+    return columns.values()
```

Comment:
how does comment come into play here?

Reply:
we would prefer the column that would have the comment,
for example if there are two columns with same name the we would pick the one which has comment

Reply:
okay